lib/evm_database.rb

```diff
@@ -16,6 +16,12 @@ class EvmDatabase
     MiqReport
     RssFeed
     MiqWidget
+    MiqAction
+    MiqEventDefinition
+    Condition
+    MiqPolicy
+    MiqPolicyContent
+    MiqPolicySet
```

Comment:
#8013 is merged, so this comment should have gone away as it was originally on PRIMORDIAL_CLASSES...the GitHub diff reorganizer relocated it to here, but this code is fine.